conbench/tests/api/test_results.py

```diff
@@ -1196,3 +1196,39 @@ def test_create_result_unexpected_stats_keys(self, client, uekey, samples):
 
         if len(samples) < 3:
             assert bmrdict["stats"][uekey] is None
+
+    def test_create_result_bad_unit(self, client):
+        self.authenticate(client)
+        result = _fixtures.VALID_RESULT_PAYLOAD.copy()
+
+        result["stats"] = {
+            "data": (3, 5),
+            "times": [],  # key must be there as of now, validate more, change this.
```

Comment:
maybe we should do this as part of this epic: remove the requirement for the `times`.

Reply:
And `time_unit`